Section 5 - Remote State Management/myrepo/kplabs-git/rds.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_db_instance" "default" {
-  allocated_storage    = 5
+  allocated_storage    = 200
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure automated backups are enabled for AWS RDS instances**
 
**Vulnerability**
Automated backups are disabled for AWS RDS instances and can cause data loss.
 
**Remediation**
Automated backups can be enabled using AWS Console or AWS CLI.
 In AWS Console - 
 1. Sign in to the AWS Console and go to the Amazon RDS console.
 2. Select Databases, and then choose the DB instance that you want to modify in the navigation pane.
 3. Select Modify.
 4. For Backup retention period, select the recommended 30 days value.
 5. Select Continue.
 6. Select Apply immediately.
 7. On the confirmation page, select Modify DB instance to save your changes and enable automated backups.
 Using AWS CLI - 
 Use the command : 'modify-db-instance' with the following parameters: 
   a.--db-instance-identifier 
   b. --backup-retention-period 
   c. --apply-immediately 
This will enable automated backups for AWS RDS instances.

source: tenablecs-